db/schema.rb

```diff
@@ -1286,6 +1286,10 @@
     t.datetime "updated_at",                  null: false
   end
 
+  create_table "tests", id: :integer, force: :cascade do |t|
+    t.text "name", null: false
+  end
+
```

Comment:
Do you need this table? tests table?

Reply:
@dawitd can you check it again please. I have made changes and pushed